compiler/src/iree/compiler/Codegen/Interfaces/BufferizationInterfaces.cpp

```diff
@@ -60,19 +60,27 @@ static Value findOrCreateSubspanBuffer(
                         .dyn_cast<IREE::Flow::DispatchTensorType>();
   assert(shapedType && shapedType.hasRank());
 
+  auto memRefType = getMemrefTypeForTensor(shapedType);
```

Comment:
Are those changes there on purpose?

Reply:
This is fixed and merged by another PR. I just rebased the code and it went away.